spec/controllers/allocation_staff_controller_spec.rb

```diff
@@ -98,17 +85,20 @@
         end
       end
 
-      context 'with an allocation' do
+      context 'when the offender has been allocated before' do
+        let!(:allocation) {
+          create(:allocation_history, prison: prison_code, nomis_offender_id: offender_no, primary_pom_nomis_id: poms.last.staff_id)
+        }
```

Comment:
```suggestion
        let!(:allocation) do
          create(:allocation_history, prison: prison_code, nomis_offender_id: offender_no, primary_pom_nomis_id: poms.last.staff_id)
        end
```

I tend to prefer `do`/`end` with multi-line `let` since this statement is too long to fit on one line.